content/chapters/compute/lab/content/user-level-threads.md

```diff
@@ -8,6 +8,15 @@ We will use the fiber implementation from libboost.
 This implementation uses a cooperative scheduler on each thread, meaning that each fiber has to yield, in order for other fiber to be executed.
 We will also use C++, and the standard `thread` implementation.
 
+## Prerequisites
+
+For this part of the lab, you will need to install `cmake` and `libboost`.
+You can do this with the following command:
+
+```console
+student@os:~$ sudo apt-get install cmake libboost-context-dev libboost-fiber-dev
```

Comment:
```suggestion
student@os:~$ sudo apt update
student@os:~$ sudo apt install cmake libboost-context-dev libboost-fiber-dev
```

Reply:
I don't have a strong opinion regarding using `apt` instead of `apt-get`, I've seen that both versions are used in this repository. Nevertheless, `update` must be run before `install`.

However, if the docker image will be updated, these lines won't be needed.

Reply:
IMHO we don't need this section since the Docker Image will be updated after this PR it's merged.

Reply:
We should keep them, in case someone runs our content natively. @cristian-vijelie add a sentence that those commands should not be run if using the Docker image.

Reply:
OK. 
@cristian-vijelie 
After this issue is resolved and the PR merge, just check if the `OS Runner Image` pipeline started.
You can ping me if any errors occur on that pipeline.